src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -77,7 +86,16 @@ export function useSwapInputsController({
   sliderXPosition: SharedValue<number>;
   slippage: SharedValue<string>;
 }) {
-  const { initialInputAmount, initialInputNativeValue } = getInitialInputValues(initialSelectedInputAsset);
+  const percentageToSwap = useDerivedValue(() => {
+    return Math.round(clamp((sliderXPosition.value - SCRUBBER_WIDTH / SLIDER_WIDTH) / SLIDER_WIDTH, 0, 1) * 100) / 100;
```

Comment:
Unrelated to this PR, but now that we have worklet math functions, does it make sense to replace `Math.round` here with `roundWorklet`